gui-wm/hyprland/hyprland-9999.ebuild

```diff
@@ -77,11 +78,11 @@ BDEPEND="
 pkg_setup() {
 	[[ ${MERGE_TYPE} == binary ]] && return
 
-	if tc-is-gcc && ver_test $(gcc-version) -lt 14 ; then
+	if tc-is-gcc && ver_test $(gcc-version) -lt 14; then
```

Comment:
Change it to the helper you used last time? ;)

Reply:
Yes I want to do it with 0.49, just want to get the slot operator in place